sql/item_latents.sql

```diff
@@ -2965,7 +2965,10 @@ INSERT INTO `item_latents` VALUES (18771,23,1,16,3);     -- ATT +1~4,party size
 INSERT INTO `item_latents` VALUES (18771,23,1,16,4);     -- ATT +1~4,party size 3+. Effect strengthens with more members
 INSERT INTO `item_latents` VALUES (18771,23,1,16,5);     -- ATT +1~4,party size 3+. Effect strengthens with more members
 INSERT INTO `item_latents` VALUES (18771,23,1,16,6);     -- ATT +1~4,party size 3+. Effect strengthens with more members
-
+-- -------------------------------------------------------
+-- Oneiros Grip
+-- -------------------------------------------------------
+INSERT INTO `item_latents` VALUES (18811,369,1,4,750);  -- Refresh MP <= 75%
```

Comment:
line break between here and the next dashed line please, for readability `-----------------`